hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobUtils.java

```diff
@@ -80,6 +59,29 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.FileNotFoundException;
```

Comment:
Import order is wrong? Please also fix the import order config. There is a import template for eclipse under the dev-support directory, if you are using IDEA please also change your config according to the template.